pkgs/applications/science/chemistry/quantum-espresso/default.nix

```diff
@@ -0,0 +1,54 @@
+{ bash
+, stdenv
+, fetchurl
+, gfortran
+, fftw
+, blas
+, liblapack
+, mpi ? null
+}:
+
```

Comment:
`bash` is now not needed anymore. Have you tried using `openblas` instead of `blas` and `lapack`? OpenBLAS is supposed to have better performance and is preferred over the reference implementations.